packages/devtools_app/lib/src/inspector/inspector_service.dart

```diff
@@ -68,6 +68,7 @@ class InspectorService extends DisposableController
     this.inspectorLibrary,
     this.supportedServiceMethods,
   ) : clients = {} {
+    // TODO(kenz): do we want to listen with event history here?
```

Comment:
@jacob314  - any input on whether we should listen to event history here?

Reply:
no need to listen here as the inspector uses a separate api to get the current selection.